tableau-databricks/connection-builder.js

```diff
@@ -30,6 +30,13 @@ limitations under the License.
 	params["SPARKSERVERTYPE"] = "3";
 	params["SSL"] = "1";
 
+	// If the HTTP path is empty, use the default HTTP path.
+	if(params["HTTPPATH"] == "") 
+	{
+		params["HTTPPATH"] = attr["default-http-path"];
```

Comment:
This line doesn't do anything right now, but if we go with Option 1, we should update the v-http-path with default-http-path if it is empty. Waiting for Tableau but not sure if we wouldn't have to do this manually like this or they will have that change